hal/hal_uv_client_usb.c

```diff
@@ -18,6 +18,7 @@
 #include "adb.h"
 #include "hal_uv_priv.h"
 #include <uv.h>
+#include <unistd.h>
```

Comment:
Is microADB include convention to include standard headers before or after project heder files?

Reply:
Different style exist.